### ISSUE TITLE
[Bug]: win2_960 Does Not Load LoRAs from Fooocus Metadata

### ISSUE THREAD

**Issue by DavidDragonsage** [metercai/SimpleSDXL#83]
### Checklist

- [X] The issue has not been resolved by following the [troubleshooting guide](https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md)
- [X] The issue exists on a clean installation of Fooocus
- [X] The issue exists in the current version of Fooocus
- [X] The issue has not been reported before recently
- [X] The issue has been reported before but has not been fixed yet

### What happened?

When loading metadata from an image created in Fooocus, SimpleSDXL did not select the specified LoRA. The first attachment shows the original image and the second attachment shows the loaded metadata. The third attachment is the regenerated image from that metadata.

![2024-08-18_11-52-45_3509](https://github.com/user-attachments/assets/1b6d9760-04bf-4c7f-ab41-703ec8fdb441)

![LoRA Described in Metadata](https://github.com/user-attachments/assets/3c11f8e5-c27b-4492-a66a-b00b1cf6b0b4)

![2024-09-23_17-20-29_2126](https://github.com/user-attachments/assets/bba9564b-2327-415d-b78c-c676dd5cf13d)

The **image log for the regenerated image** shows that the LoRA was used but that was not the case:
{
"prompt": "a field of wildflowers. Trees and a beautiful Afro-Canadian woman in the sharp background, deep depth of field, clearly **defined** facial features, (rim light:1.4)",
"negative_prompt": "blurry background, bokeh",
"prompt_expansion": "a field of wildflowers. Trees and a beautiful Afro-Canadian woman in the sharp background, deep depth of field, clearly defined facial features, (rim light:1.4), dazzling warm illumination, detailed, intricate, elegant, highly contrasted, dramatic professional, cinematic, artistic, great composition, dynamic, rich vibrant colors, striking, attractive, aesthetic, very",
"styles": "['Fooocus V2', 'Fooocus Enhance']",
"performance": "Speed",
"steps": 30,
"resolution": "(896, 1152)",
"guidance_scale": 7,
"sharpness": 6,
"adm_guidance": "(1.5, 0.8, 0.3)",
"base_model": "SDXLFaetastic_v24.safetensors",
"refiner_model": "None",
"refiner_switch": 0.6,
"clip_skip": 2,
"sampler": "dpmpp_2m_sde_gpu",
"scheduler": "karras",
"vae": "Default (model)",
"seed": "5419787220761092607",
"lora_combined_1": "sd_xl_offset_example-lora_1.0 : 0.5",
"backend_engine": "SDXL-Fooocus",
"metadata_scheme": "fooocus",
"version": "Fooocus v2.5.5 SimpleSDXL_v20240916.fe3afb6"
}

When I looked at the list of LoRAs under the Models tab, no LoRA was actually checked off. In this example the sd_xl_offset_example-lora should have been checked. See the fourth attachment.

![LoRA Not Selected](https://github.com/user-attachments/assets/8449aba0-f5fb-49c6-9b45-d378eebf4bd9)

I checked the LoRA manually, see the fifth attachment.

![LoRA Selected Manually](https://github.com/user-attachments/assets/accf97a9-a0e1-47f5-ac37-c3275d08d0fa)

Then I pressed Generate and got the correct image. I know from experience with this specific image that this the correct SimpleSDXL interpretation of it, although there are some differences from the original. See the last attachment.

![2024-09-23_17-56-14_2126](https://github.com/user-attachments/assets/0340015e-84bc-446e-8b75-0d9aef21d5e8)

**Image Log, LoRA Loaded Manually**
{
"prompt": "a field of wildflowers. Trees and a beautiful Afro-Canadian woman in the sharp background, deep depth of field, clearly defined facial features, (rim light:1.4)",
"negative_prompt": "blurry background, bokeh",
"prompt_expansion": "a field of wildflowers. Trees and a beautiful Afro-Canadian woman in the sharp background, deep depth of field, clearly defined facial features, (rim light:1.4), dazzling warm illumination, detailed, intricate, elegant, highly contrasted, dramatic professional, cinematic, artistic, great composition, dynamic, rich vibrant colors, striking, attractive, aesthetic, very",
"styles": "['Fooocus V2', 'Fooocus Enhance']",
"performance": "Speed",
"steps": 30,
"resolution": "(896, 1152)",
"guidance_scale": 7,
"sharpness": 6,
"adm_guidance": "(1.5, 0.8, 0.3)",
"base_model": "SDXLFaetastic_v24.safetensors",
"refiner_model": "None",
"refiner_switch": 0.6,
"clip_skip": 2,
"sampler": "dpmpp_2m_sde_gpu",
"scheduler": "karras",
"vae": "Default (model)",
"seed": "5419787220761092607",
"lora_combined_1": "sd_xl_offset_example-lora_1.0.safetensors : 0.5",
"backend_engine": "SDXL-Fooocus",
"metadata_scheme": "fooocus",
"version": "Fooocus v2.5.5 SimpleSDXL_v20240916.fe3afb6"
}

### Steps to reproduce the problem

1) Find an image created in Fooocus that was created using a LoRA.
2) Load that image into the metadata pane and Apply Metadata
3) Generate a new image
4) Compare the new image to the original. There will be a noticeable difference.
5) Select the Models tab and examine the LoRA list. No LoRA will be checked off.
6) Manually select the LoRA that was used by the original image.
7) Generate another new image.
8) Compare this final image to the original. The images will be very similar if not identical.

### What should have happened?

SimpleSDXL2 should have created an image that was identical or almost identical to the original image.

### What browsers do you use to access Fooocus?

Mozilla Firefox

### Where are you running Fooocus?

Locally

### What operating system are you using?

Windows 10

### Console logs

```Shell
Console Log, Lora file not found:

App started successful. Use the app with http://192.168.1.69:8186/ or 192.168.1.69:8186
[Fooocus] GPU memory: max_reserved=2.080GB, max_allocated=1.973GB, reserved=2.080GB, free=8.903GB, free_torch=0.107GB, free_total=9.011GB, gpu_total=12.000GB, torch_total=2.080GB
[ToolBox] Reset_params_from_image: -->SDXL-Fooocus params from the image with embedded parameters.
[Fooocus] GPU memory: max_reserved=2.080GB, max_allocated=1.973GB, reserved=2.080GB, free=8.903GB, free_torch=0.107GB, free_total=9.011GB, gpu_total=12.000GB, torch_total=2.080GB
[TaskEngine] Task_class:Fooocus, Task_name:default, Task_method:text2image
[TaskEngine] Enable Fooocus backend.
[Comfyd] Comfyd freeing!
[Parameters] Adaptive CFG = 7
[Parameters] CLIP Skip = 2
[Parameters] Sharpness = 6
[Parameters] ControlNet Softness = 0.25
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] Seed = 5419787220761092607
[Parameters] CFG = 7
[Fooocus] Loading control models ...
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 30 - 18
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
model_type EPS
UNet ADM Dimension 2816
Using xformers attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using xformers attention in VAE
extra {'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_l.text_projection'}
left over keys: dict_keys(['cond_stage_model.clip_l.transformer.text_model.embeddings.position_ids'])
Base model loaded: F:\SimpleAI\SimpleModels\checkpoints\SDXLFaetastic_v24.safetensors
VAE loaded: None
Request to load LoRAs [('sd_xl_offset_example-lora_1.0', 0.5)] for model [F:\SimpleAI\SimpleModels\checkpoints\SDXLFaetastic_v24.safetensors].
Lora file not found: F:\SimpleAI\SimpleModels\loras\sd_xl_offset_example-lora_1.0
Requested to load SDXLClipModel
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.96 seconds
[Fooocus] Processing prompts ...
[Wildcards] Copmile text in prompt to arrays: a field of wildflowers. Trees and a beautiful Afro-Canadian woman in the sharp background, deep depth of field, clearly defined facial features, (rim light:1.4) -> arrays:[], mult:0
[Fooocus] Preparing Fooocus text #1 ...
F:\SimpleAI\SimpleSDXL2_win_0916\python_embeded\lib\site-packages\transformers\models\gpt2\modeling_gpt2.py:650: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at ..\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:455.)
  attn_output = torch.nn.functional.scaled_dot_product_attention(
[Prompt Expansion] a field of wildflowers. Trees and a beautiful Afro-Canadian woman in the sharp background, deep depth of field, clearly defined facial features, (rim light:1.4), dazzling warm illumination, detailed, intricate, elegant, highly contrasted, dramatic professional, cinematic, artistic, great composition, dynamic, rich vibrant colors, striking, attractive, aesthetic, very
[Fooocus] Encoding positive #1 ...
[Fooocus Model Management] Moving model(s) has taken 0.18 seconds
[Fooocus] Encoding negative #1 ...
[Parameters] Denoising Strength = 1.0
[Parameters] Initial Latent shape: Image Space (1152, 896)
Preparation time: 200.25 seconds
Using karras scheduler.
[Fooocus] GPU memory: max_reserved=2.111GB, max_allocated=1.992GB, reserved=2.006GB, free=8.946GB, free_torch=0.270GB, free_total=9.216GB, gpu_total=12.000GB, torch_total=2.006GB
[Fooocus] Preparing Fooocus task 1/1 ...
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 2.36 seconds
100%|██████████████████████████████████████████████████████████████████████████████████| 30/30 [00:25<00:00,  1.19it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 2.45 seconds
[Fooocus] Saving image 1/1 to system ...
Image generated with private log at: E:\stable-diffusion-webui\outputs\2024-09-23\log.html
Generating and saving time: 32.56 seconds
[Fooocus] GPU memory: max_reserved=5.662GB, max_allocated=5.218GB, reserved=3.393GB, free=7.540GB, free_torch=3.213GB, free_total=10.753GB, gpu_total=12.000GB, torch_total=3.393GB
[Enhance] Skipping, preconditions aren't met
Processing time (total): 32.56 seconds
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
Total time: 232.85 seconds


Console Log, LoRA Loaded after Manual Selection

[Fooocus] GPU memory: max_reserved=3.393GB, max_allocated=1.992GB, reserved=2.145GB, free=8.788GB, free_torch=0.152GB, free_total=8.940GB, gpu_total=12.000GB, torch_total=2.145GB
[TaskEngine] Task_class:Fooocus, Task_name:default, Task_method:text2image
[TaskEngine] Enable Fooocus backend.
[Comfyd] Comfyd freeing!
[Parameters] Adaptive CFG = 7
[Parameters] CLIP Skip = 2
[Parameters] Sharpness = 6
[Parameters] ControlNet Softness = 0.25
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] Seed = 5419787220761092607
[Parameters] CFG = 7
[Fooocus] Loading control models ...
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 30 - 18
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
Request to load LoRAs [('sd_xl_offset_example-lora_1.0.safetensors', 0.5)] for model [F:\SimpleAI\SimpleModels\checkpoints\SDXLFaetastic_v24.safetensors].
Loaded LoRA [F:\SimpleAI\SimpleModels\loras\sd_xl_offset_example-lora_1.0.safetensors] for UNet [F:\SimpleAI\SimpleModels\checkpoints\SDXLFaetastic_v24.safetensors] with 788 keys at weight 0.5.
Requested to load SDXLClipModel
Loading 1 new model
unload clone 1
[Fooocus Model Management] Moving model(s) has taken 1.29 seconds
[Fooocus] Processing prompts ...
[Wildcards] Copmile text in prompt to arrays: a field of wildflowers. Trees and a beautiful Afro-Canadian woman in the sharp background, deep depth of field, clearly defined facial features, (rim light:1.4) -> arrays:[], mult:0
[Fooocus] Preparing Fooocus text #1 ...
[Prompt Expansion] a field of wildflowers. Trees and a beautiful Afro-Canadian woman in the sharp background, deep depth of field, clearly defined facial features, (rim light:1.4), dazzling warm illumination, detailed, intricate, elegant, highly contrasted, dramatic professional, cinematic, artistic, great composition, dynamic, rich vibrant colors, striking, attractive, aesthetic, very
[Fooocus] Encoding positive #1 ...
[Fooocus] Encoding negative #1 ...
[Parameters] Denoising Strength = 1.0
[Parameters] Initial Latent shape: Image Space (1152, 896)
Preparation time: 2.73 seconds
Using karras scheduler.
[Fooocus] GPU memory: max_reserved=2.145GB, max_allocated=2.002GB, reserved=2.020GB, free=8.913GB, free_torch=0.273GB, free_total=9.186GB, gpu_total=12.000GB, torch_total=2.020GB
[Fooocus] Preparing Fooocus task 1/1 ...
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 4.37 seconds
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████| 30/30 [00:23<00:00,  1.26it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.18 seconds
[Fooocus] Saving image 1/1 to system ...
Image generated with private log at: E:\stable-diffusion-webui\outputs\2024-09-23\log.html
Generating and saving time: 29.64 seconds
[Fooocus] GPU memory: max_reserved=5.486GB, max_allocated=5.152GB, reserved=3.420GB, free=7.513GB, free_torch=3.229GB, free_total=10.742GB, gpu_total=12.000GB, torch_total=3.420GB
[Enhance] Skipping, preconditions aren't met
```

### Additional information

This issue was first reported as **win2_960 Does Not Load LoRAs from Metadata #82**

**Comment by metercai** [comment]
这里是SimpleSDXL不是Fooocus，虽然是衍生版，但有自己的独立性。
This is SimpleSDXL, not Fooocus. Although it is a derivative version, it has its own independence
